patches/tModLoader/Terraria/Item.cs.patch

```diff
@@ -918,6 +925,15 @@
  			if (ItemID.Sets.BossBag[type])
  				return Color.Lerp(newColor, Color.White, 0.4f);
  
+@@ -47734,7 +_,7 @@
+ 				int num11 = FindDecraftAmount();
+ 				Recipe recipe = Main.recipe[decraftingRecipeIndex];
+ 				int num12 = 0;
+-				bool flag = recipe.requiredItem[1].stack > 0;
++				bool flag = recipe.requiredItem.Count > 1 && recipe.requiredItem[1].stack > 0;
```

Comment:
I think just `recipe.requiredItem.Count > 1` is correct here. This flag looks to be checking for multiple ingredients.